java/src/main/java/com/training/db/hw2/Factorial.java

```diff
@@ -0,0 +1,38 @@
+package com.training.db.hw2;
+
+public class Factorial {
+
+    public static void calculateFactorial(int n2_fac) {
+
+        int n3 = UiUtil.input("Input third number: ");
+        int i = 1, result = 1;
+
+        switch (n2_fac){
+            case 1:
+                while (i <= n3) {
+                    result = result * i;
```

Comment:
here you can use *= operator i.e. result *= i;

Reply:
done